workflows/illumina.nf

```diff
@@ -78,15 +79,21 @@ workflow ILLUMINA {
     ch_assemblies = METASPADES.out.assembly
     ch_versions = ch_versions.mix(METASPADES.out.versions)
 
-    // run QUAST on assemblies for stats
+    // rename contigs after assembly for cleaner downstream steps
+    RENAME_CONTIGS (
+        ch_assemblies, "metaspades"
+    )
+    reformatted_assemblies = RENAME_CONTIGS.out.reformatted_assembly
```

Comment:
mm yeah since it's an output channel, I can't remember if in my nanopore workflow I'm consistent about channel names or just do `PROCESS.out` to feed between steps